index.js

```diff
@@ -4,8 +4,8 @@ var isHash = ref.isHash
 var isFeedId = ref.isFeedId
 var timestamp = require('monotonic-timestamp')
 var isCanonicalBase64 = require('is-canonical-base64')
-var isEncryptedRx = isCanonicalBase64('','\.box.*')
-var isSignatureRx = isCanonicalBase64('','\.sig.\w+')
+var isEncryptedRx = isCanonicalBase64('','\\.box.*')
+var isSignatureRx = isCanonicalBase64('','\\.sig.\\w+')
```

Comment:
Why are these changes needed?

Reply:
They're totally broken. :upside_down_face: 

We want `/\.box.*/`, but the previous code accidentally gave us `/.box.*/` because the backslash is interpreted as an escape character and ignored (because `.` doesn't need to be escaped in JavaScript strings. The `.` gives us a greedy match that gets a bunch of characters and actually breaks the regex. If you run either of these regexes, they **always** return `false` because they fail to match. It's just that we have *another* bug that's been masking the issue.

(I'll comment on the other bug.)

Reply:
wow, nice find :-)